pyartcd/pyartcd/tekton.py

```
@@ -0,0 +1,148 @@
+            "params": [
+                {"name": k, "value": str(v)} for k, v in merged_params.items() if v is not None and str(v) != ""
+            ],
+        },
+    }
+
+    pr_json = json.dumps(pipeline_run)
+    logger.info("Creating PipelineRun for pipeline %s in namespace %s", pipeline_name, namespace)
+    logger.debug("PipelineRun spec: %s", pr_json)
+
+    result = subprocess.run(
+        ["oc", "create", "-f", "-"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)